homeassistant/components/sensor/plex.py

```diff
@@ -32,6 +34,8 @@
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
     vol.Optional(CONF_PASSWORD): cv.string,
     vol.Optional(CONF_TOKEN): cv.string,
+    vol.Optional(CONF_SSL, default=False): cv.boolean,
+    vol.Optional(CONF_VERIFY_SSL, default=True): cv.boolean,
```

Comment:
Why would we add this?

Reply:
Likely I thought they were required for some reason... I'll test without.